zulipterminal/model.py

```diff
@@ -906,11 +907,35 @@ def _handle_typing_event(self, event: Event) -> None:
                 else:
                     raise RuntimeError("Unknown typing event operation")
 
-    def get_invalid_recipient_emails(self, recipient_emails: List[str]
-                                     ) -> List[str]:
-
-        return [email for email in recipient_emails
-                if email not in self.user_dict]
+    def get_invalid_recipients(self, recipient_text: str) -> List[str]:
```

Comment:
You said this commit is WIP so I'm guessing we'll have a test in a later iteration :)

Reply:
Absolutely, once we decide on how we want to handle this, I'll add relevant tests.